vms/event/tests/test_model.py

```diff
@@ -37,6 +37,7 @@ def test_valid_model_create(self):
         self.assertEqual(event_in_db.name, event[0])
         self.assertEqual(str(event_in_db.start_date), event[1])
         self.assertEqual(str(event_in_db.end_date), event[2])
+        self.assertEqual(str(event_in_db.description), event[3])
```

Comment:
Why do you need `str` here? Isn't it a charfield already?

Reply:
@anjali-dhanuka you don't need this function as it's already a textfield.

Reply:
Ok @kamsuri I'll remove it.